modules/logging/efk/README.md

```diff
@@ -1,3 +1,35 @@
-This chart installs an elasticsearch cluster with kibana, elastic-curator and filebeat by default. You can optionally enable logstash and install Fluentd if you prefer. It also optionally installs nginx-ldapauth-proxy.
+## Elastic Kibana Filebeat
 
-[Elastick-stack Documentation](https://github.com/helm/charts/tree/master/stable/elastic-stack)
+https://helm.elastic.co/
+
+
+## Prerequisite
```

Comment:
is it possible to use this module without Google OAuth keys?

Reply:
I fixed it. If we don't set efk_oauth_domain - deploy kibana without google auth